tests/Database/DatabaseEloquentMorphToTest.php

```diff
@@ -378,8 +379,9 @@ public function getRelation($parent = null, $builder = null)
         $this->builder = $builder ?: m::mock(Builder::class);
         $this->builder->shouldReceive('where')->with('relation.id', '=', 'foreign.value');
         $this->related = m::mock(Model::class);
-        $this->related->shouldReceive('getKeyName')->andReturn('id');
+        $this->related->shouldReceive('getcolumn')->andReturn('id');
```

Comment:
Is this supposed to be `getColumn`?

Reply:
doesn't really matter. method names are not case sensitive.